sensor/tests/resource/helper.go

```diff
@@ -606,10 +614,14 @@ func GetLastMessageWithDeploymentName(messages []*central.MsgFromSensor, ns, nam
 	return lastMessage
 }
 
-// GetLastAlertsWithDeploymentID find most recent alert message by deployment ID
-func GetLastAlertsWithDeploymentID(messages []*central.MsgFromSensor, id string) *central.MsgFromSensor {
+// GetLastAlertsWithDeploymentID find most recent alert message by deployment ID, If checkEmptyAlerts is set to true it will also check for AlertResults with no Alerts
```

Comment:
```suggestion
// GetLastAlertsWithDeploymentID find most recent alert message by deployment ID. If checkEmptyAlerts is set to true it will also check for AlertResults with no Alerts
```

Reply:
👍